test/it/pipeline/src/test/java/org/apache/shardingsphere/test/it/data/pipeline/core/job/service/GovernanceRepositoryAPIImplTest.java

```diff
@@ -87,6 +87,8 @@ void assertIsJobConfigurationExisted() {
         assertFalse(governanceRepositoryAPI.isJobConfigurationExisted("foo_job"));
         getClusterPersistRepository().persist("/pipeline/jobs/foo_job/config", "foo");
         assertTrue(governanceRepositoryAPI.isJobConfigurationExisted("foo_job"));
+        getClusterPersistRepository().delete("/pipeline/jobs/foo_job/config");
+        assertFalse(governanceRepositoryAPI.isJobConfigurationExisted("foo_job"));
```

Comment:
`/pipeline/jobs/foo_job/config` might be not deleted if assertion failure before deleting. And registry center (embedded ZK) is shared for now, the invalid job config is still might be get in other places.

Could we persist minimal and valid job config in IT?